crates/block-producer/src/state_view/tests/verify_tx.rs

```diff
@@ -228,3 +229,80 @@ async fn test_verify_tx_vt5() {
         Err(VerifyTxError::InputNotesAlreadyConsumed(vec![nullifier_in_both_txs]))
     );
 }
+
+/// Tests that `verify_tx()` succeeds when the unauthenticated input note found in the in-flight notes
+#[tokio::test]
+#[miden_node_test_macro::enable_logging]
+async fn test_verify_tx_in_flight_notes() {
```

Comment:
Nit, though these are getting fairly long..
```suggestion
async fn test_verify_tx_unauthenticated_inflight_notes_are_accepted() {
```